modules/email.js

```diff
@@ -0,0 +1,31 @@
+const nodemailer = require("nodemailer");
+
+const { M_USER, M_PASS } = process.env;
+
+const transporter = nodemailer.createTransport({
+  host: "smtp.mailgun.org",
+  port: 587,
+  secure: false,
+  auth: {
+    user: M_USER,
+    pass: M_PASS,
+  },
+});
+
+const sendEmail = async ({ to, subject, html }) => {
+  const mailOptions = {
+    from: '"Szkola Node" <szkola@gmail.com>',
+    to,
+    subject,
+    html,
```

Comment:
Jak masz czas to możesz zrobić ładnie generowany html np za pomocą handlebars. Albo statyczny kod html który bedzie elegancko wyglądał w przeglądarce.